dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -537,17 +537,17 @@
 
     public static final String ON_DISCONNECT_KEY = "ondisconnect";
 
-    public static final String ON_INVOKE_METHOD_KEY = "oninvoke.method";
+    public static final String ON_INVOKE_METHOD_KEY = "oninvokeMethod";
```

Comment:
I think the original value is more reasonable since it's used in URL.

Reply:
agree with beiwei30

Reply:
Actually, those constants value is not references any more. Formerly, it was used for the mapping between annatation of parameter and the key of Map. Now, this mapping is no need any more.